### ISSUE TITLE
Consistency warns when passing a function as a parameter

### ISSUE THREAD

**Issue by rhnonose** [rrrene/credo#418]
### Precheck

Similar problem to #370

### Environment

* Credo version: `0.8.4`
* Erlang/Elixir version: `Erlang/OTP 20, Elixir 1.5.0`
* Operating system: `macOS Sierra 10.12.5`

### What were you trying to do?

I'm passing a function from a module attributed to a constant:
```elixir
defmodule CredoTest do
  @moduledoc false
  defmodule Test1 do
    @moduledoc false
    def test(_), do: :ok
  end

  @module Test1

  def hello do
    [1, 2, 3]
    |> Enum.map(&@module.test/1)
  end
end
```

### Expected outcome

No credo Warnings

### Actual outcome

```
┃ [C] ↗ There are spaces around operators most of the time, but not here.
┃       lib/credo_test.ex:12:30 #(CredoTest.hello)
```

**Comment by rrrene** [comment]
@rhnonose Thx for reporting this! :+1: It should be be fixed in the latest version of Credo.

If it is not, please feel free to re-open this issue!
